VedmidOleksandra/HW5/task_5.3.py

```diff
@@ -0,0 +1,15 @@
+# Task3. Write a script that will calculate the factorial of the entered number without using recursion.
+# Example: 0!=1, 1!=1, 2!=2, 3!= 1*2*3=6, ....
+
```

Comment:
good job
look at this implementation
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)